step_definitions/adapters/adapter_login_steps.py

```diff
@@ -14,13 +14,22 @@
 }
 EXTENSION_NAME = "Five9 Agent Desktop Toolkit"
 
+def check_force_login(agent):
+    try:
+        common.wait_element_to_be_clickable(ADAPTER_LOGIN_PAGE.driver,ADAPTER_LOGIN_PAGE.force_login_button,5)
+        ADAPTER_LOGIN_PAGE.get_force_login_button().click()
+        return
+    except:
```

Comment:
let's try to catch the exceptions individually. this way we can debug faster when some error happen.

Reply:
Fixed